cmd/localCI/README.rst

```diff
@@ -22,9 +22,9 @@ Logging
 
 By default **localCI** places all the logs generated by each stage in ``/var/log/localCI/`` inside a directory with the
 same number of the pull request, but you can specify a different path modifying ``logDir`` in the `Configuration file`_.
-Two files with the suffix ``.stdout`` and ``.stderr`` are generated by stage. For example if the ``logDir`` is
+One file with the same name of the stage is generated per each stage. For example if the ``logDir`` is
```

Comment:
Some missing commas and a suggestion for re-wording.
Could read: 
One log file per stage is generated with the same name. For example, if the ``logDir`` is ``/var/log/localCI`` and the pull request being tested is the number ``53``, when the stage ``setup`` is run, the full path to its log file would be ``/var/log/localCI/53/setup``.